server/py/hangman.py

```diff
@@ -83,7 +86,14 @@ def apply_action(self, action: GuessLetterAction) -> None:
 
     def get_player_view(self, idx_player: int) -> HangmanGameState:
```

Comment:
In der `get_player_view` Methode gibt es den Parameter `idx_player`. Ich denke, dieser ist falsch für den Hangman use-case, da es ein Einspieler-Spiel ist. Damit man diesen Parameter aber nicht immer passen muss, wenn man die Method `get_player_view` aufruft, würde ich diese Zeile forlgendermassen ändern:

```suggestion
    def get_player_view(self, idx_player: Optional[int] = None) -> HangmanGameState:
```